<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonPush">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Returned Book</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bookInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="stackListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="queueListWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Returned Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPeekStack">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Latest Returned Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnqueue">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reserve Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDequeue">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Serve Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPeekQueue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Next Reservation</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Library Management System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stacklabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stack Operations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Queue Operations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recently Returned Books</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reservationInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/library.png);
background-position: center;
background-repeat: no-repeat;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButtonPush</zorder>
   <zorder>bookInput</zorder>
   <zorder>stackListWidget</zorder>
   <zorder>queueListWidget</zorder>
   <zorder>pushButtonPop</zorder>
   <zorder>pushButtonPeekStack</zorder>
   <zorder>pushButtonEnqueue</zorder>
   <zorder>pushButtonDequeue</zorder>
   <zorder>pushButtonPeekQueue</zorder>
   <zorder>statusLabel</zorder>
   <zorder>titleLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>stacklabel</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>reservationInput</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
